code/modules/modular_armor/modular.dm

```diff
@@ -544,20 +559,27 @@
 		return TRUE
 
 	attachment.do_detach(user, src)
-	update_overlays()
+	update_icon()
 	return TRUE
 
 
 /obj/item/clothing/head/modular/update_overlays()
 	. = ..()
+	/*
 	if(installed_module)
-		. += image(installed_module.icon, installed_module.item_state)
+		. += image(installed_module.icon, ITEM_STATE_IF_SET(installed_module))*/ //these look whack because they dont have non-singl direction spritos.
```

Comment:
commented out because they dindt work before and if they work now they'll go whack when sprite change direction.